docs/deployment/README.md

```diff
@@ -39,27 +39,41 @@ The process involves:
 
 #### Install Terraform
 
-If you are using a Mac and [Homebrew](https://brew.sh), installing Terraform is
+It is recommended to keep a consistent version of Terraform as you deploy. Once your state files are migrated to a higher version, they are not always backwards compatible so integrators should pin their Terraform version. This is easily accomplished using the Terraform Version Manager [tfenv](https://github.com/tfutils/tfenv). If you have a CI environment (or any other machine) that you are using to deploy, **make sure that your local Terraform version is at least equal to the version on CI**, otherwise you will run into errors trying to re-deploy from your local machine.
```

Comment:
```suggestion
It is recommended to keep a consistent version of Terraform as you deploy. Once your state files are migrated to a higher version, they are not always backwards compatible so integrators should pin their Terraform version. This is easily accomplished using the Terraform Version Manager [tfenv](https://github.com/tfutils/tfenv). If you have a CI environment (or any other machine) that you are using to deploy the same stack, **you should pin your version across those machines as well*, otherwise you will run into errors trying to re-deploy from your local machine.
```

It seemed like making the advice consistent to always use pinning was more clear